### PR TITLE
🪿 Update Canadian info 🇨🇦

### PR DESCRIPTION
**Pull request by djensenius**
I'm messing around with Canadian Kia EV status.

I noticed some Canadian things were incomplete or broken. I'm fixing these. It's a bit hacky as I had to make my own interface for some data that's returned, but I tried to work with existing datatypes as best I could.

Thoughts?

### CODE REVIEW

File: src/interfaces/common.interfaces.ts [hunk 1]
```diff
@@ -399,6 +399,7 @@ export interface VehicleOdometer {
 export interface VehicleStatusOptions {
   refresh: boolean;
   parsed: boolean;
+  useInfo?: boolean;
```
Comment (Hacksore) [inline src/interfaces/common.interfaces.ts:402]: This is really only used for CA region from what I could tell and I guess ok for now.

File: debug.ts [hunk 2]
```diff
@@ -33,6 +33,11 @@ let client: BlueLinky;
 let vehicle;
 const { username, password, pin } = config;
 
+let { useInfo } = config;
```
Comment (Hacksore) [inline debug.ts:36]: You could always

```ts
const { username, password, pin, useInfo = false } = config;
```

**Review by Hacksore — CHANGES_REQUESTED** [review]
Once you address the feedback this can be merged.

**Review by Hacksore — APPROVED** [review]

**Review by Hacksore — CHANGES_REQUESTED** [review]
Sorry was in the wrong tab in hit approve 😬

File: debug.ts (hunk 2 above)
Comment (djensenius) [reply debug.ts:36]: Can't use const, but can let! Didn't know I could do this, thanks! :)

File: Session.vim
[the remark this answers is not in the document]
Comment (djensenius) [reply Session.vim:1]: 👋

**Comment by djensenius** [comment]
Okay! How's it looking now?

File: debug.ts (hunk 2 above)
Comment (djensenius) [reply debug.ts]: Wait, I lied, fixed.

**Comment by Hacksore** [comment]
Will get around to this sometime this week.

**Review by Hacksore — APPROVED** [review]

**Pull request merged**